### PR TITLE
Split the AuthenticatesAndRegisterUsers into two

### PR DESCRIPTION
**Pull request by MadsLundholmDk**
Changes made on the basis of this pull request from laravel/framework: https://github.com/laravel/framework/pull/7903

### CODE REVIEW

**Comment by conorsmith** [comment]
You changed the application's namespace from `App` to `BlueCMS`.

You also removed the `@return` annotation from the constructors' docblocks, which would be out of the scope of this PR (and is probably just a merging accident).

**Comment by MadsLundholmDk** [comment]
Whoops. Changed these things back.

File: app/Http/Controllers/Auth/AuthController.php [hunk 1]
```diff
@@ -18,7 +20,7 @@ class AuthController extends Controller {
 	|
 	*/
 
-	use AuthenticatesAndRegistersUsers;
+    use RedirectPaths, AuthenticatesUsers, RegistersUsers;
```
Comment (RomainLanz) [inline app/Http/Controllers/Auth/AuthController.php]: Wrong indentation

File: app/Http/Controllers/Auth/PasswordController.php [hunk 2]
```diff
@@ -18,7 +19,7 @@ class PasswordController extends Controller {
 	|
 	*/
 
-	use ResetsPasswords;
+    use RedirectPaths, ResetsPasswords;
```
Comment (RomainLanz) [inline app/Http/Controllers/Auth/PasswordController.php]: Same

**Pull request closed without merging**